sdks/python/apache_beam/transforms/resources.py

```diff
@@ -38,11 +39,26 @@ def _parse_int(value):
   return str(value).encode('ascii')
 
 
+def _return_max(v1, v2):
```

Comment:
This and the pair of "method" dicts makes me wonder if this'd be better expressed as a mapping of urns to ResourceHint subclasses, each of which knows how to parse, merge, and represent itself as a byte string.

Reply:
Agreed.